test/blackbox-tests/test-cases/ppx-cross-context-issue.t/run.t

```diff
@@ -1,12 +1,29 @@
-# Using a ppx in a cross-compiled build context makes dune try to build the ppx
-# in the target context instead of the host, then fail.
-  $ dune build --debug-dependency-path
-  File "lib/dune", line 3, characters 18-24:
-  3 |  (preprocess (pps fooppx)))
-                        ^^^^^^
-  Error: Library "fooppx" in _build/cross-environment/ppx is hidden
-  (unsatisfied 'enabled_if').
-  -> required by _build/cross-environment/lib/lib.pp.ml
-  -> required by alias lib/all (context cross-environment)
-  -> required by alias default (context cross-environment)
-  [1]
+Dune uses the host context to look up dependencies and build PPXes
+
+  $ dune build --display=short
```

Comment:
In general, we should avoid `--display=short` in the tests. Its output changes with many arbitrary changes. I think the fact that the artifacts are built should be enough here.

Reply:
That's fair. I find it very useful to check the shortened build log. I'll try to remember to omit it in future changes.